packages/substrate/.env.test

```diff
@@ -1 +1 @@
-SYGMA_ENV="testnet"
\ No newline at end of file
+SYGMA_ENV="local"
```

Comment:
why the change here? just curious 🤔

Reply:
Mostly for the mocked config to be returned from file config.ts in core package:

```ts
  async init(environment: Environment): Promise<void> {
    if (environment === Environment.LOCAL) {
      this.configuration = localConfig;
```